src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -1088,6 +1089,168 @@ async def redirect_uri(request: Request):
+                        client = getattr(group_instance, "_mcp_client", None)
+                        if client is None:
+                            raise RuntimeError(f"MCP client not found for group {group_name}")
+
+                        try:
+                            session_healthy = False
+                            server_tools: dict[str, Any] = {}
+
+                            try:
+                                server_tools = await client.get_tools()
+                                session_healthy = True
+                            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)